### PR TITLE
build(deps): Upgrade highlightjs to 9.18.1

### PR DESCRIPTION
**Pull request by ConradJChan**
Tested against the Sanity folder for all browsers

### CODE REVIEW

File: src/third-party/text/2.64.0/remarkable.min.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/*! remarkable 1.7.1 https://github.com/jonschlinkert/remarkable @license MIT */
```
Comment (jstoffan) [inline src/third-party/text/2.64.0/remarkable.min.js:1]: Looks like this is the only asset we don't have in a non-minified state. Does that matter?

**Review by jstoffan — APPROVED** [review]
Should the commit message be more general, since we're updating multiple libraries?

**Comment by ConradJChan** [comment]
This commit is just for highlightjs, I'll do separate commits for remarkable and papaparse

**Comment by jstoffan** [comment]
Gotcha. The assets copied over from the prior version bamboozled me. LGTM

**Pull request merged**